plugins/CMakeLists.txt

```diff
@@ -30,6 +31,9 @@ endif (WITH_history)
 if (WITH_history2)
   add_subdirectory( history2 )
 endif (WITH_history2)
+if (WITH_history3)
```

Comment:
Or you can also name it `ChatHistory` similar to what you have like `ChatHistoryHandler`